### ISSUE TITLE
Priority 1 acessibility issue: Decorative image mark up is incorrect for assistive technology

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6539]
**What**

Non-text content is used for pure decoration but is not implemented in a way that it can be ignored by assistive technology.

**WCAG Reference:**
1.1.1 Non-text Content - Level A
Understanding Non-text Content | How to Meet Non-text Content

**Current Code Ref(s):** 

#main > article > div.Space-sc-1ntp1c3-0.eEdhXm > div > div:nth- child(3) > div > div > div > img

**Solution:**
Text alternatives should provide a literal description of the image content except where the image is purely decorative. To ensure that decorative images are ignored by assistive technology, make sure that the alt attribute is included and has a null value.

**Example:**
 <img width="3200" height="1800" class="lazy-image bg-charcoal font-white image lazyloaded" [...] alt="" [...]>

For more information, please refer to F38: Failure of Success Criterion 1.1.1 due to not marking up decorative images in HTML in a way that allows assistive technology to ignore them

<img width="3200" height="1800" class="lazy-image bg-charcoal font-white image lazyloaded" [...] alt="Photograph of the entrance to a museum showing a large poster with the text, 'Hello there, come on in' written in large letters. To the left of the sign is a smiling female security guard wearing a transparent visor on her head. To the right of the sign in the distance is a large banner on the wall showing a photograph of a man seated in a wheelchair next to a large mannequin artwork. Under the photograph are the words, 'Being Human'. " [...]>

**Where**

This is a global issue

 
<img width="818" alt="Screenshot 2021-05-27 at 10 50 29" src="https://user-images.githubusercontent.com/7315126/119805691-61079680-bed9-11eb-8d9b-1d7678db33d4.png">

**Comment by gestchild** [comment]
I was curious about this one. I'm not sure I agree that this image is pure decoration.

I thought the images intention is to set a welcoming tone and the alt text description does that too, "... 'Hello there, come on in' written in large letters. To the left of the sign is a smiling female security guard..."

If we were to remove the alt text, so assistive technologies ignore it, we need to establish if this is always the case for header images.

**Comment by dannybirchall** [comment]
I think I'm with Gareth on querying this -- we choose engaging people-centred images to be welcoming, and I think that this meaning might carry across.

An additional complication is that you can only set the alt attribute on an image once in Prismic -- so if there are multiple instances of the image and not all are 'decorative' we could have a problem.

**Comment by gestchild** [comment]
Having discussed this with Sarah Parry from Digital Accessibility Centre and with various members of the team, I think we're confident that these images are not decorative and therefore the alt text is appropriate to include.
